test/repl/proxysql_repl_tests/bin/mysql-data-checksum.bash

```
@@ -30,17 +30,16 @@ res=1
 
 echo "Running checksum for sysbench data on primary (master) ..."
 echo "Checksum results:"
-mysql ${SSLOPT} -h${MYSQL1_HOST}${INFRA} -P${MYSQL1_PORT} -uroot -proot sysbench --skip-column-names -e " \
+mysql -h${MYSQL1_HOST}${INFRA} -P${MYSQL1_PORT} -uroot -proot sysbench --skip-column-names -e " \
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)